contentcuration/contentcuration/static/less/content-container.less

```diff
@@ -273,8 +303,10 @@
 	    width: -o-max-content;
 		width: -moz-max-content;
 		width: max-content;
-    	padding-right: 200px;
-    	list-style: none;
+		list-style: none;
+		display: grid;
+		grid-template-columns: repeat(1000, auto);
```

Comment:
There might be a better way of doing this...  I just wanted to make sure that the topic tree never wraps, so this is meant to give "unlimited" columns.